CMakeLists.txt

```diff
@@ -1,21 +1,9 @@
-cmake_minimum_required(VERSION 2.8.7)
-if(POLICY CMP0022)
-  cmake_policy(SET CMP0022 NEW)
-endif()
-if(POLICY CMP0025)
-  cmake_policy(SET CMP0025 NEW)
-endif()
-if(POLICY CMP0026)
-  cmake_policy(SET CMP0026 NEW)
-endif()
-if(POLICY CMP0042)
-  cmake_policy(SET CMP0042 NEW)
-endif()
-if( POLICY CMP0063 )
-  cmake_policy(SET CMP0063 NEW)
-endif()
+cmake_minimum_required(VERSION 3.8.2 FATAL_ERROR)
```

Comment:
This is a no-go for me. I use Travis to build GDCM, currently only Ubuntu Trusty image are available, cmake version is 2.8.12